arduino/sensors/linefollower/SensorArduino.ino

```diff
@@ -1,5 +1,6 @@
 #include <Wire.h>
 #include "sensorbar.h"
+#include "smoothing.h" //not sure how we're including with the make file
```

Comment:
I think we have to use the `-libraries` flag. I'll figure it out later.

Reply:
Where do you get "sensorbar.h"? is that a 3rd party install?

Reply:
Ya it's the library for the sensor bar. I'll see if I can add it in.

Reply:
Can it be installed via the Arduino library manager? I just might need to install them.

Reply:
I don't know I think I just downloaded it and placed it in the libraries folder.
https://github.com/sparkfun/SparkFun_Line_Follower_Array_Arduino_Library